querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -176,6 +177,14 @@ export const DataDocScheduleForm: React.FunctionComponent<
                 const notificationField = (
                     <>
                         <FormSectionHeader>Notification</FormSectionHeader>
+                        <div>
+                            Notification will be sent to the owner of the
```

Comment:
they don't need to be an owner, i think the editor of datadoc can also make scheduled datadoc, in that case, the notifications are sent to them